### ISSUE TITLE
Quasar Error on startup - Raspberry - Kodi 17.1 - OpenElec 8.0.3

### ISSUE THREAD

**Issue by hollisterca** [scakemyer/plugin.video.quasar#845]
It just sais Quasar Error. see Log.
I turned on debug logging and tried again.
See attachment
[01_Kodi1.log.zip](https://github.com/scakemyer/plugin.video.quasar/files/992114/01_Kodi1.log.zip)

**Comment by elgatito** [comment]
http://wiki.openelec.tv/index.php/OpenELEC_Configuration_Addon#tab=Network
"Wait for network before starting Kodi" -> turn on, and try again.

**Comment by hollisterca** [comment]
I will try that later. Problem is that when I try to access the settings - OpenELEC, it tells me "Settings addon is not yet ready, please try again later"

**Comment by elgatito** [comment]
Please, provide a kodi.log with debug enabled. Full log, not a cut with only error place.
http://kodi.wiki/view/Log_file/Easy

**Comment by hollisterca** [comment]
Here is the kodi.log 
So I booted the Rasp Pi on May 18, enabled logging, clicked on Quasar, had the error, but nothing is in the kodi.log for today.
[02_System.log.zip](https://github.com/scakemyer/plugin.video.quasar/files/1011021/02_System.log.zip)

**Comment by elgatito** [comment]
Still it's not a debug log. 
Don't see any error. Only once when user cancelled the buffering.

**Comment by hollisterca** [comment]
Here the uploaded log
https://paste.ubuntu.com/24603648/

**Comment by elgatito** [comment]
```
10:22:30.000 T:1538257824   DEBUG: [plugin.video.quasar] Requesting http://localhost:65251/ from ['plugin://plugin.video.quasar/', '1', '']
10:22:30.068 T:1538257824   ERROR: [plugin.video.quasar] Traceback (most recent call last):
10:22:30.068 T:1538257824   ERROR: [plugin.video.quasar]   File "/storage/.kodi/addons/plugin.video.quasar/resources/site-packages/quasar/navigation.py", line 178, in run
10:22:30.069 T:1538257824   ERROR: [plugin.video.quasar]     data = _json(url)
10:22:30.070 T:1538257824   ERROR: [plugin.video.quasar]   File "/storage/.kodi/addons/plugin.video.quasar/resources/site-packages/quasar/navigation.py", line 125, in _json
10:22:30.070 T:1538257824   ERROR: [plugin.video.quasar]     with closing(urllib2.urlopen(url)) as response:
10:22:30.070 T:1538257824   ERROR: [plugin.video.quasar]   File "/usr/lib/python2.7/urllib2.py", line 154, in urlopen
10:22:30.071 T:1538257824   ERROR: [plugin.video.quasar]   File "/usr/lib/python2.7/urllib2.py", line 429, in open
10:22:30.071 T:1538257824   ERROR: [plugin.video.quasar]   File "/usr/lib/python2.7/urllib2.py", line 447, in _open
10:22:30.072 T:1538257824   ERROR: [plugin.video.quasar]   File "/usr/lib/python2.7/urllib2.py", line 407, in _call_chain
10:22:30.072 T:1538257824   ERROR: [plugin.video.quasar]   File "/usr/lib/python2.7/urllib2.py", line 1228, in http_open
10:22:30.073 T:1538257824   ERROR: [plugin.video.quasar]   File "/usr/lib/python2.7/urllib2.py", line 1198, in do_open
10:22:30.074 T:1538257824   ERROR: [plugin.video.quasar] URLError: <urlopen error [Errno -2] Name or service not known>
```

Kodi cannot open `http://localhost:65251/`, this is very strange and still looks like a system problem. Firewall/network settings on the system maybe.

You can open in a browser this url: http://IP_OF_KODI_DEVICE:65251/ , just insert IP there.

**Comment by hollisterca** [comment]
seems to work
![urlok](https://cloud.githubusercontent.com/assets/23412806/26240915/149b057a-3c83-11e7-85d9-2ff45a4a2689.jpg)

PS: If I pop in the other mem card with Openelec 7.0.1 everything works. (same Rasp. Pi, same IP)

**Comment by elgatito** [comment]
```
08:22:07.823 T:1946886160  NOTICE: Local hostname: OE8
```

From the Rpi shell try to run: `ping localhost`
and `ping OE8`

**Comment by hollisterca** [comment]
How do I get to the shell? ssh is not enabled and I can not get to that screen.

**Comment by elgatito** [comment]
http://openelec.tv/forum/12-guides-tips-and-tricks/887-guide-guide-for-ssh-within-openelec#21249

Anyway, that is a problem with your network settings. Quasar is working.

**Comment by hollisterca** [comment]
on your link it shows you on the very end: 
The ssh_enable hack has not been supported since OE 3.0

**Comment by hollisterca** [comment]
Resolved with OpenElec 8.0.4
